### PR TITLE
New version: Hugo.Hugo.Extended version 0.123.5

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/141325)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-02-28T18:44:19.4848546Z ##[error] Manifest Error: Required field missing. [NestedInstallerFiles]

(Automated response - build 847.)

File: manifests/h/Hugo/Hugo/Extended/0.123.5/Hugo.Hugo.Extended.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created with komac v2.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Hugo.Hugo.Extended
+PackageVersion: 0.123.5
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles: []
```
Comment (stephengillie) [inline manifests/h/Hugo/Hugo/Extended/0.123.5/Hugo.Hugo.Extended.installer.yaml:8]:
```suggestion
NestedInstallerFiles:
- RelativeFilePath: hugo.exe
  PortableCommandAlias: hugo
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Copied from [version 0.123.3](https://github.com/microsoft/winget-pkgs/pull/140765/files).

**Comment by stephengillie** [comment]
Spaces detected in version number.

(Automated response - build 853)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**